nall/intrinsics.hpp

```diff
@@ -173,16 +173,21 @@ namespace nall {
   struct Architecture {
     static constexpr bool x86   = 1;
     static constexpr bool amd64 = 0;
+    static constexpr bool sse42 = 0;
     static constexpr bool arm64 = 0;
     static constexpr bool arm32 = 0;
     static constexpr bool ppc64 = 0;
     static constexpr bool ppc32 = 0;
   };
 #elif defined(__amd64__) || defined(_M_AMD64)
   #define ARCHITECTURE_AMD64
+  #ifndef __SSE4_1__
+  #define __SSE4_1__ 0
+  #endif
   struct Architecture {
     static constexpr bool x86   = 0;
     static constexpr bool amd64 = 1;
+    static constexpr bool sse41 = __SSE4_1__;
```

Comment:
This is `sse41` while all the other `Architecture` structs have `sse42`?

Reply:
Right I need to fix it

Reply:
i already tried compiling whit the ss41 and the ss42 lines but its crashing both ways

Reply:
Is it crashing or not compiling?

There is a problem in the patch as shown by the CI. I'll fix it. Afterwards, when it compiles for you, it should work

Reply:
The problem should be fixed now. Please try again.